cruise-control/src/yaml/endpoints/rightsize.yaml

```diff
@@ -0,0 +1,89 @@
+RightsizeEndpoint:
+  post:
+    operationId: rightsize
+    summary: Test the provisioner on Azure
+    parameters:
+      - name: num_brokers_to_add
+        in: query
+        description: The difference in broker count to rightsize towards.
+        schema:
+          type: integer
+          format: int32
+          default: -1
+      - name: partition_count
+        in: query
+        description: The target number of partitions to rightsize towards.
+        schema:
+          type: integer
+          format: int32
+          default: -1
```

Comment:
Note / reminder: the defaults might change based on the previous feedback in this PR.